### PR TITLE
Annotations.

### PR DESCRIPTION
**Pull request by jaeilepp**
Added annotations block + fixed wrong tags for epochs selection and drop log.

### CODE REVIEW

File: DictionaryTags_MNE.txt [hunk 1]
```diff
@@ -175,11 +175,13 @@ mne_rt_command              3700 ? "realtime command"
 mne_rt_client_id            3701 ? "realtime client"
 
 #
-# 3710... MNE epochs bookkeeping
+# 3800... MNE epochs bookkeeping
```
Comment (larsoner) [inline DictionaryTags_MNE.txt:178]: so did Elekta actually give us the 3800- range and this was wrong before?

**Comment by agramfort** [comment]
merge if you're convinced it's good :)

File: DictionaryTags_MNE.txt (hunk 1 above)
Comment (jaeilepp) [inline DictionaryTags_MNE.txt:178]: I suppose the range for MNE is 3500-3999 as is said at the top of this file. These values are used in `mne.io.constants`, so they were just wrong here.

Comment (larsoner) [inline DictionaryTags_MNE.txt:178]: ahh, makes sense. Feel free to merge if you're happy

**Pull request merged**